docs/pages/getting-started/linux-server.mdx

```diff
@@ -206,15 +214,15 @@ $ tsh ls
 $ tsh ssh root@node-name
 ```
 
-### Add a node to the cluster
+### Add a Node to the cluster
```

Comment:
If you're going to capitalize node in this title, you might want to capitalize cluster as well.

Reply:
See my note re: "Node" above